src/main/steps/respondent/hub-page/content.ts

```diff
@@ -377,6 +377,11 @@ const cy: typeof en = ({ isDivorce, partner, userCase, contactEmail }: CommonCon
     line4: 'apply for Help With Fees here',
     link: 'https://www.gov.uk/get-help-with-court-fees',
   },
+  finalOrderComplete: {
+    line1: 'Your final order has been granted. You are now legally divorced.',
+    line2: `<a class="govuk-link" href="/downloads/final-order-granted" download="Final-Order-Granted">Download a copy of your ‘final order’</a>.
```

Comment:
same

Reply:
Moved the links to nunjucks files and renamed references